### ISSUE TITLE
Implement new protocol proposal stages

### ISSUE THREAD

**Issue by jellegerbrandy** [daostack/subgraph#81]
Conversation from discord:

```A question about the stages (perhaps @tsuberim  remembers) in the proposal in the subgraph
This code:
  let state = gpProposal.value2;
  if (state === 1) {
    // Closed
    proposal.stage = 'Resolved';
  } else if (state === 2) {
    // Executed
    proposal.stage = 'Resolved';
  } else if (state === 3) {
    // PreBoosted
    proposal.stage = 'Open';
  } else if (state === 4) {
    // Boosted
    proposal.stage = 'Boosted';
  } else if (state === 5) {
    // QuietEndingPeriod
    proposal.stage = 'QuietEndingPeriod';
  }
I do not get this. whey is the  origianl state simplified into a stage in which we basically loose info?
Or, more specifically: I want tog et the list of preboosted proposals - I'd like to filter by stage: 'PreBoosted' or something similar.
OSToday at 1:14 PM
This code does not support the new protocol update .
There is a differentiation between Qued ,PreBoosted ,Boosted and Expired proposal.
tsuberimToday at 1:15 PM
You also have the executed (or similar) field to know if it's resolved and closed or resolved and executed
OSToday at 1:15 PM
On latest infra I also add an event for the state switch ...(which was missing in some cases),```

**Comment by jellegerbrandy** [comment]
And while we're at it we should also implent the executionstate.

Also "stage" should be renamed "state" for consistency.

**Comment by jellegerbrandy** [comment]
Atcthe moment this ia blocked by https://github.com/daostack/migration/issues/51 which makes subgraph tests fail

**Comment by jellegerbrandy** [comment]
This is being worked in on PR #85

**Comment by orenyodfat** [comment]
fix in pr #85
